packages/s/sof-firmware/pspec_x86_64.xml

```diff
@@ -21,32 +21,67 @@
 </Description>
         <PartOf>kernel</PartOf>
         <Files>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic-2ch.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic-4ch.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic-ace1-2ch.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic-ace1-4ch.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic-cavs25-2ch.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic-cavs25-4ch.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic-idisp.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-hda-generic.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-cs42l43-l0-cs35l56-l12.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-es83x6-ssp1-hdmi-ssp02.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-hdmi-ssp02.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-max98357a-rt5682-ssp2-ssp0-2ch-pdm1.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-max98357a-rt5682-ssp2-ssp0.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-max98357a-rt5682.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt1019-rt5682.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt1318-l12-rt714-l0.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt711-4ch.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt711-l0-rt1316-l23-rt714-l1.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt712-l0-rt1712-l3.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt713-l0-rt1316-l12-rt1713-l3.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt713-l0-rt1316-l12.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-rt722-l0.tplg</Path>
-            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg/sof-mtl-sdw-cs42l42-l0-max98363-l2.tplg</Path>
+            <Path fileType="data">/lib/firmware/intel/sof-ace-tplg</Path>
```

Comment:
Were these moved or removed upstream? Looks like we're just installing an empty directory now.